udine1/index.html

```diff
@@ -10,9 +10,146 @@
+	<link rel="stylesheet" href="assets/font-awesome/font-awesome/css/font-awesome.min.css" type="text/css">
   </head>
 
   <body>
+  
+	<!-- Parallax
+      ================================================== -->
+	  
+		<div class="container-fluid">
+			<div class="row parallax">
+				<div class="col-lg-4 col-md-4 col-xs-12">
+					<section class="help">
+					<img src="img/icon_1_parallax.png">
+					<h3>Hai bisogno di aiuto?</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam lectus risus</p>
+					<button class="btn light-blue">Centro assistenza</button>
+					</section>
+				</div>
+				<div class="col-lg-4 col-md-4 col-xs-12">
+					<section class="riservata">
+					<img src="img/icon_2_parallax.png">
+					<h3>La tua area riservata</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam lectus risus</p>
+					<button class="btn light-blue">Area riservata</button>
+					</section>
+				</div>
+				<div class="col-lg-4 col-md-4 col-xs-12">
+					<section class="seleziona">
+					<img src="img/icon_3_parallax.png">
+					<h3>LOREM dove vuoi tu</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam lectus risus</p>
+					<input class="seleziona_input" type="text" name="seleziona" id="seleziona" size="30" placeholder="Seleziona il paese"><i class="glyphicon glyphicon-search margin-negative"></i><br>
+					</section>
+				</div>
+			</div>
+		</div>
+  
+	<!-- I nostri Contatti & Newsletter
+      ================================================== -->
+	
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-lg-6 col-md-6 col-xs-12 no-margin">
+					<section class="contatti">
+					<h2 class="cheading">I nostri contatti</h2>
+					<div class="glyphicon glyphicon-map-marker" style="color:#0090d2;"></div><div class="content"> Via Lorem ipsum - 33100 Lorem (UD) Italy</div><br>
+					<div class="glyphicon glyphicon-earphone" style="color:#0090d2;"></div><div class="content"> T. +39 0000 00000 / F. +39 0000 00000</div><br>
+					<div class="glyphicon glyphicon-envelope" style="color:#0090d2;"></div> E. <div class="content"><a href="mailto:lorem@loremipsum.it">lorem@loremipsum.it</a></div>
+					</section>
+				</div>
+				<div class="col-lg-6 col-md-6 col-xs-12 no-margin">
+					<section class="newsletter">
+					<h2 class="nheading">Newsletter</h2>
+					<input class="newsletter_input" type="text" name="letter" id="letter" size="46" placeholder="Il tuo indirizzo email..."><br>
+					<input class="newsletter_checkbox" type="checkbox" id="accetto"><p> accetto il trattamento dei dati personali</p>
+					<input class="newsletter_button" type="button" name="button" id="button" value="Iscriviti">
+					</section>
+				</div>
+			</div>
+		</div>
+	  
+	<!-- footer
+      ================================================== -->
+	<footer class="margin-top-40">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-3 col-md-3 col-xs-6">
```

Comment:
`class="col-lg-3 col-md-3 col-xs-6"` is redundant and can be simplified to `class="col-xs-6 col-md-3"`